server-side/framework/php/README.md

```diff
@@ -0,0 +1,5 @@
+# PHP
+
+## Frameworks for PHP
+
+* Laravel
```

Comment:
Can you make this into a link to the README.md in the laravel dir?
example:
[Laravel](laravel/README.md)

Reply:
Yes I can, but that was copied from the python readme so you might want to update that one as well.

Reply:
Actually the Node one needs to be updated as well